pkg/dns/dns_test.go

```diff
@@ -207,21 +207,9 @@ func TestSkyPodHostnameSRVLookup(t *testing.T) {
 
 	service := newHeadlessService()
 	endpointIPs := []string{"10.0.0.1", "10.0.0.2"}
-	endpoints := newEndpoints(service, newSubsetWithOnePort("", 80, endpointIPs...))
-
-	// The format of thes annotations is:
-	// endpoints.beta.kubernetes.io/hostnames-map: '{"ep-ip":{"HostName":"pod request hostname"}}'
-	epRecords := map[string]endpointsapi.HostRecord{}
-	for i, ep := range endpointIPs {
-		epRecords[ep] = endpointsapi.HostRecord{HostName: fmt.Sprintf("ep-%d", i)}
```

Comment:
I guess Tim means https://github.com/kubernetes/kubernetes/blob/master/pkg/api/endpoints/util.go#L38-L41 and https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/endpoints/util.go#L38-L41.